teamB/eungSooKim/app.js

```diff
@@ -53,11 +49,118 @@ app.post("/users", async (req, res, next) => {
             password
         ) VALUES (?, ?, ?);
         `,
-    [name, email, password]
+    [name, email, passWord]
   );
   res.status(201).json({ message: "userCreated" });
 });
 
+app.post("/posts/write/:userId", async (req, res) => {
+  const { title, content } = req.body;
+  const userId = req.params.userId;
+
+  await database.query(
+    `INSERT INTO posts(
+            title,
+            content,
+            user_id
+        ) VALUES (?, ?, ${userId});
+        `,
+    [title, content]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+app.get("/posts/all", async (req, res) => {
+  await database.query(
+    `SELECT posts.user_id AS userId,
+                users.profile_image AS userProfileImage,
+                posts.id AS postingId,
+                posts.title AS postingTitle,
+                posts.profile_image AS potingImageUrl,
+                posts.content As postingContent
+         FROM users JOIN posts ON users.id = posts.user_id`,
+    (err, rows) => {
+      res.status(200).json(rows);
+    }
+  );
+});
+
+
+app.get("/users/posts/:userId", async (req, res) => {
+  let userId = req.params.userId;
+  const result = {};
+
+  await database.query(
+    `SELECT users.id AS userId,
+            users.profile_image AS userProfileId
+         FROM users WHERE users.id = ${userId}`,
+    (err, rows) => {
+      result.userId = rows[0].userId
+      result.userProfileId = rows[0].userProfileId;
+      database.query(
+        `SELECT posts.id AS postingID, 
+        posts.profile_image AS postingImageUrl, 
+        posts.title AS postingContent FROM posts 
+        WHERE ${userId} = posts.user_id`,
+        (err, postInfo) => {
+          result.postings = postInfo
+          res.status(200).json({data:result});
+        }
+      );
+    }
+  );
+});
+
+app.put("/posts/modify/:postId", async(req, res) => {
```

Comment:
```suggestion
app.put("/posts/:postId", async(req, res) => {
```
put method 자체에 Modify의 의미를 담고 있습니다.